<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Factures</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher une facture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>551</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Afficher</string>
      </property>
      <widget class="QTableView" name="tab_fact">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>481</width>
         <height>271</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>140</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background: #FFFFFF;
border: 2px solid #FFD700;
border-bottom-color: #FFD700; /* same as the pane color */
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 120px;
padding: 8px;
color: #696969;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QPushButton:hover {
color:#fff;
background:#FFD700;
border: none;
}
</string>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter une facture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>551</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QLineEdit" name="le_id_f">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
 border-width: 3px;
border-style: solid;
border-color: #ccfff0;
  border-radius: 10px;
padding:0 8;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_date_f">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>220</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ttc_f">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id Facteur :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>220</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Facteur :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix Facteur :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>310</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-width : 1px;
border-style : outset;
border-color:rgb(60,170,153);
}
QPushButton:hover {
color:#fff;
background:#24a0ed;
border: none;
}
</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer une facture</string>
     </attribute>
     <widget class="QToolButton" name="supp">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_supp_f">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>491</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>supp</zorder>
     <zorder>le_supp_f</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier une factures</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Facteur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id Facteur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix Facteur :</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Hash-tech</string>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
